backend/dataall/modules/datasets/cdk/dataset_stack.py

```diff
@@ -28,6 +29,8 @@
 
 logger = logging.getLogger(__name__)
 
+GACCO_ALLOW_CROSS_ACCOUNT_TAG = ("gacco:AllowPublicAccess", "True")
```

Comment:
what exactly is this tag used for (I see it is attached to the dataset KMS key but what is the reason) ?

Reply:
also coming from merge conflict. Removed this as its not needed.